### DISCUSSION TITLE
测试数学公式

### DISCUSSION THREAD

**Discussion by share121** [share121/inter-knot#1333] [其他]
$y=x^2+5$

$$
y=\frac{1}{x}
$$

**Comment by VacuolePaoo** [comment]
[xss.pdf](https://github.com/user-attachments/files/16571730/xss.pdf)

**Comment by VacuolePaoo** [comment]
![xss](https://github.com/user-attachments/assets/3e980f3a-4bc6-4904-830a-1779e2b27a85)

**Comment by share121** [comment]
……

**Comment by su2569** [comment]
额...
